pkgs/top-level/python-packages.nix

```
@@ -3314,6 +3314,10 @@ in {
 
   kitchen = callPackage ../development/python-modules/kitchen { };
 
+  kivy = callPackage ../development/python-modules/kivy { inherit (pkgs) mesa; };
```

```suggestion
  kivy = callPackage ../development/python-modules/kivy {
    inherit (pkgs) mesa;
  };
```

This is more common across nixpkgs.

---

Applied.